src/classes/CRLP_Operation.cls

```diff
@@ -102,12 +110,18 @@ public class CRLP_Operation {
      * @return CRLP_Operation.RollupType enum
      */
     public static CRLP_Operation.RollupType getRollupTypeFromString(String operation) {
-        for (CRLP_Operation.RollupType rt : getRollupOperationTypes().keySet()) {
-            if (rt.name() == operation) {
-                return rt;
+        if (rollupOperationTypesByName == null) {
+            rollupOperationTypesByName = new Map<String, CRLP_Operation.RollupType>();
+            for (CRLP_Operation.RollupType rt : getRollupOperationTypes().keySet()) {
+                // Need multiple revisions of the same operation type to ensure the map.get() returns a value.
+                // Example: Best_Year != BEST_YEAR != 'Best Year' -- as a result all variations needed as the Map key
```

Comment:
Good point. There are only 2 calls to this method, though I kind of like the idea of having the method/map handle the complicated logic of normalizing the key. This way the caller only needs to pass a value rather than have to deal with normalizing that value.